core/src/main/java/jenkins/model/NewViewLink.java

```diff
@@ -44,8 +44,7 @@ public String getDisplayName() {
 
             @Override
             public String getUrlName() {
-                String urlName = Jenkins.getInstance().getRootUrl() + URL_NAME;
-                return urlName;
+                return "/" + URL_NAME;
```

Comment:
👍

Reply:
This seems a bug to begin with. Does this link even work within folders?

Reply:
@daniel-beck Thanks for bringing this up, it was reported as JENKINS-51291 and so I put the reference here as well.

For the record, here are the links computed in a folder context

<details>

<summary>Screenshot</summary>

![view_global_vs_local](https://user-images.githubusercontent.com/2662497/55725071-1ed71e80-5a0d-11e9-88c4-23b451083ee4.png)

</details>

Reply:
Hmm will this break if Jenkins is not the root context?

Reply:
The whole action should probably be removed until someone has time to do it properly.

Reply:
> Hmm will this break if Jenkins is not the root context?

Just double checked right now, it works with both `/jenkins` and `/`

> The whole action should probably be removed until someone has time to do it properly.

The scope of this PR is to reduce the usage of rootURL, the "bug behavior" is not changed at all here.

Reply:
It seems like this change should work fine. But, what exactly does it correct or improve? Does it fix [JENKINS-51291]?
I don't object to changing this line but I'm not understanding how the change improves anything. As Daniel suggests, it might be better to fix the action.